src/annotator/test/sidebar-test.js

```diff
@@ -1138,4 +1138,14 @@ describe('Sidebar', () => {
       assert.calledWith(guestRPC().call, 'selectAnnotations', tags, true);
     });
   });
+
+  describe('createSidebarIframe', () => {
+    it('does not let `allow` attribute to be overwritten', () => {
+      const iframe = createSidebarIframe({ sidebarAppUrl: 'https://foo.com' });
+
+      assert.throws(() => {
+        iframe.allow = 'something else';
+      }, "Cannot assign to read only property 'allow' of object '#<HTMLIFrameElement>'");
+    });
```

Comment:
I note in Via this exception doesn't get thrown. I'm not sure if this because the JS that attempts to write to the `allow` property is not executing in strict mode, or if Via is silently swallowing the exception. If the issue is the former, this could potentially break in future, in that case we could fix the issue by modifying the client to add a no-op setter which does nothing (and maybe emits a warning).

Reply:
Yeah, I thought about this, but finally decided not to worry too much about it because the upstream project accepted this was not an intended behavior and we will contribute a fix there.

Hopefully, we will be able to remove this workaround in a couple months, but we should definitely keep an eye on it.

I'm also planning to thoroughly test this in QA, just in case strict mode is less strict just because it's localhost, but we end up seeing this error in other envs.

Reply:
Ok, it works in QA now, so proceeding to prod.